### DISCUSSION TITLE
Option to save/load world in MP

### DISCUSSION THREAD

**Discussion by CleitonDeLima** [diasurgical/DevilutionX#4142] [Ideas]
About the issue #74 

It would be interesting to save the progress of the multiplayer game. 
About my case:
Me and my friend would like to finish the game but we don't have much time to finish the game in one day.

It would be interesting the idea of being able to save the progress as it is done in single player.

One idea is to have a configuration in the `diablo.ini` even though it could enable the progress save for multiplayer, in which case the host could save the game, when loading a saved game the other players would enter a game that already exists.

I don't know if this is actually possible, it's more of an idea :).

**Comment by qndel** [comment]
What's the point of your message? You literally repeated what https://github.com/diasurgical/devilutionX/issues/74 was about.

**Reply by CleitonDeLima** [reply]
I thought new ideas could only be commented here and not in issue. :smile:

**Comment by AJenbo** [comment]
It's both possible and planned, see #74 :)
